src/solidity-frontend/solidity_language.cpp

```diff
@@ -0,0 +1,133 @@
+/*******************************************************************\
+
+Module: Solidity AST module
+
+\*******************************************************************/
+
+#include <solidity-frontend/solidity_language.h>
+#include <solidity-frontend/solidity_convert.h>
+
+languaget *new_solidity_language(const messaget &msg)
+{
+  return new solidity_languaget(msg);
+}
+
+solidity_languaget::solidity_languaget(const messaget &msg) : languaget(msg)
+{
+  clang_c_module = new_clang_c_language(msg);
+}
+
+solidity_languaget::~solidity_languaget()
+{
+  if(clang_c_module != nullptr)
```

Comment:
you probably don't need this here.

Reply:
Done. 
Changed to `assert` to make sure we have valid `clang_c_module`.